lib/AFS/CellCC/Restore.pm

```diff
@@ -471,10 +471,18 @@ _do_release($$) {
     my $pid = spawn_child(name => 'vos release handler',
                           stderr => $stderr_fh->filename,
                           cb => sub {
+        my %args;
+        my %flags = %{ config_get('vos/flags/release') };
         my $vos = vos_auth();
-        $vos->release(id => $job->{volname},
-                      cell => $job->{dst_cell})
-        or die("vos release error: ".$vos->errors());
+
+        $args{id} = $job->{volname};
+        $args{cell} = $job->{dst_cell};
+
+        while (my ($key, $value) = each %flags) {
+            $args{$key} = $value;
```

Comment:
Maybe, I should call supportsArgument (from afs-command) here.

Reply:
I don't think that's needed, unless the failure mode is really bad/confusing when AFS::Command gets an unrecognized argument. What happens now?

Reply:
The failure mode is not confusing (we get a message like: "Unsupported arguments"). So, yes, I think you are correct.